resources/services/glue/security_configurations.hcl

```diff
@@ -0,0 +1,42 @@
+service          = "aws"
+output_directory = "."
+add_generate     = true
+
+
+description_modifier "remove_read_only" {
+  words = ["  This member is required."]
+}
+
+resource "aws" "glue" "security_configurations" {
+  path = "github.com/aws/aws-sdk-go-v2/service/glue/types.SecurityConfiguration"
+  ignoreError "IgnoreAccessDenied" {
+    path = "github.com/cloudquery/cq-provider-aws/client.IgnoreAccessDeniedServiceDisabled"
+  }
+  deleteFilter "AccountRegionFilter" {
+    path = "github.com/cloudquery/cq-provider-aws/client.DeleteAccountRegionFilter"
+  }
+  multiplex "AwsAccountRegion" {
+    path   = "github.com/cloudquery/cq-provider-aws/client.ServiceAccountRegionMultiplexer"
+    params = ["glue"]
+  }
+  options {
+    primary_keys = ["account_id", "name"]
```

Comment:
This is just a question, because I don't know: do security configurations names necessarily have to be unique account-wide? Or should `region` also be part of the primary key, maybe?